classes/controller/Controller.php

```diff
@@ -176,7 +180,11 @@ public function init()
             define('_PS_BASE_URL_SSL_', Tools::getShopDomainSsl(true));
         }
 
-        $this->container = $this->buildContainer();
+        if (null === $this->container) {
```

Comment:
I'm not sure this check is useful, as this method should be called only one time